packages/panzer/disc-fe/src/Panzer_FaceToElement.hpp

```diff
@@ -78,10 +78,19 @@ class FaceToElement {
 
   FaceToElement(panzer::ConnManager & conn);
 
-  /** Build the mapping from a mesh topology.
+  FaceToElement(panzer::ConnManager & conn,
+                const Teuchos::RCP<const Teuchos::Comm<int>> comm);
+
+  /** Build the mapping from a mesh topology using MPI_COMM_WORLD.
     */
   void initialize(panzer::ConnManager & conn);
```

Comment:
Are you maintaining this for backwards compatiblity?

Reply:
Lets do that, just to keep things clean.

Reply:
OK.  Since it looks like the autotester is backed up, I will deprecate it now.